components/brave_vpn/brave_vpn_service.cc

```diff
@@ -129,8 +120,8 @@ BraveVpnService::BraveVpnService(
       skus_service_getter_(skus_service_getter),
       api_request_helper_(GetNetworkTrafficAnnotationTag(),
                           url_loader_factory) {
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
We're safe to move this down a line (and do the `DCHECK` on `IsBraveVPNEnabled` on Android)

Reply:
done